pom.xml

```diff
@@ -64,6 +64,7 @@
 		<skipTests>false</skipTests>
 		<spring-cloud-build.version>3.0.4</spring-cloud-build.version>
```

Comment:
Should you remove this then?

Reply:
No, we are still using it for the checkstyle plugin. This will go away soon -- I am thinking the best time to do the checkstyle changes is right before forking off Spring Cloud GCP 2.x (with Spring Boot 2.5 support) and leaving the main branch to be Spring Cloud GCP 3.x (with Spring Boot 2.6 support).